assets/sass/okta-theme.scss

```diff
@@ -81,8 +81,10 @@
 }
 
 #okta-sign-in.auth-container {
-  // for IE to support main tag's width
-  display: block;
+  @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```

Comment:
curious why this change?

Reply:
Remember we had a conversation on slack with Pradeep about applying this css to IE only because it was only needed for IE

Reply:
So it applies only when it's IE

Reply:
Good to put a screenshot from chrome or firefox as well..

Reply:
added a screenshot for Chrome

Reply:
> So it applies only when it's IE ?

Yes

Reply:
cool.. It seems this is something we missed in last HF we did..

Reply:
It was your ask post hotfix. Am I ok to merge this in or do I need to wait for the monthly release? It's not a bugfix or FF gated, so I believe that means monthly release, but this is transparent to users.

Reply:
yeah you can merge to master.. it's SIW, it's anyway going to be released later.